editor/src/clj/editor/gui.clj

```diff
@@ -2549,6 +2553,7 @@
   (inherits outline/OutlineNode)
   (input names g/Str :array)
   (output name-counts NameCounts :cached (g/fnk [names] (frequencies names)))
+  (output texture-resource-names GuiResourceNames :cached (g/fnk [names] (into (sorted-set) names)))
```

Comment:
Used to check ownership of texture gui resources when renaming such a gui resource in a referenced scene.